### PR TITLE
community: bug fix - replace image uris with b64_texts

### PR DESCRIPTION
**Pull request by masands**
- **Description:** Fix an issue where the image path (uris) were inserted instead of the b64 texts when image metadata is present. In response to https://github.com/langchain-ai/langchain/issues/15935
  - **Issue:** Returns b64 text instead of image uris when doing a similarity search with multimodal embeddings

### CODE REVIEW

**Review by baskaryan — COMMENTED** [review]
cc @jeffchuber

File: libs/community/langchain_community/vectorstores/chroma.py [hunk 1]
```diff
@@ -209,7 +209,7 @@ def add_images(
                     empty_ids.append(idx)
             if non_empty_ids:
                 metadatas = [metadatas[idx] for idx in non_empty_ids]
-                images_with_metadatas = [uris[idx] for idx in non_empty_ids]
+                images_with_metadatas = [b64_texts[idx] for idx in non_empty_ids]
```
Comment (baskaryan) [inline libs/community/langchain_community/vectorstores/chroma.py:212]: do we need to do this in L234 as well

Comment (jeffchuber) [reply libs/community/langchain_community/vectorstores/chroma.py:212]: is this for multimodality?

**Comment by ccurme** [comment]
Closing as author is unresponsive, will re-open if needed.

**Pull request closed without merging**